test-suite/djinni/test.djinni

```diff
@@ -1,3 +1,9 @@
+@extern "../../support-lib/future.yaml"
+
+async_interface = interface +j +o +w {
```

Comment:
why snake case here for async_interface?  These are normally Pascal Case, eg AsyncInterface

Reply:
all the other djinni test interfaces are snake_case

Reply:
my question is oriented at the general case

Reply:
The convention is use snake_case in djinni idl files.  The generated code style is defined by the command line, eg `--ident-jni-class NativeFooBar`.